activiti-spring-security-policies/src/test/java/org/activiti/spring/security/policies/RuntimeTestConfiguraiton.java

```diff
@@ -17,28 +17,27 @@
 public class RuntimeTestConfiguraiton {
 
     @Bean
-    public UserDetailsService myUserDetailsService() {
```

Comment:
@pancudaniel7 you changed this again?? It needs to be UserDetailsService which is a spring class.